lib/matplotlib/rcsetup.py

```diff
@@ -110,7 +110,7 @@ def validate_float(s):
 
 def validate_float_or_None(s):
     """convert s to float or raise"""
-    if s is None:
+    if s is None or s == 'None':
```

Comment:
Because the object from an rcfile can't be a `None` object, only the string. There is only one situation that I can find where it is valid to say None in the rc file and mean the object and one situation where it is valid to say None/none in the rc file and mean the string or possibly the object (the default interpolation mode). This takes care of the former situation. We probably should double-check how we are handling the latter.